src/js/common/components/NavButtons.jsx

```diff
@@ -174,8 +174,8 @@ export default class NavButtons extends React.Component {
       );
     } else if (path.endsWith('introduction')) {
       buttons = (
-        <div className="row form-progress-buttons">
```

Comment:
`form-progress-buttons` gave it `max-width: 47rem`, but the `small` breakpoint > 47rem, so there was this weird gap where the button was really long, but didn't take up the whole width.